src/vm/exceptionhandling.cpp

```diff
@@ -199,6 +200,7 @@ static inline void UpdatePerformanceMetrics(CrawlFrame *pcfThisFrame, BOOL bIsRe
 {
     WRAPPER_NO_CONTRACT;
     COUNTER_ONLY(GetPerfCounters().m_Excep.cThrown++);
+    g_exceptionCount++;
```

Comment:
Why do we need a new one here? This is already track immediately above by `COUNTER_ONLY(GetPerfCounters().m_Excep.cThrown++);`

Reply:
I believe `COUNTER_ONLY` is no-op, left over from desktop. We should delete it since it is not coming back - we are introducing alternative mechanism.

Reply:
```COUNTER_ONLY``` is used everywhere -- I'll just submit a separate PR that cleans this stuff out later.

Reply:
@jkotas `COUNTER_ONLY` may be a nop, but what about `GetPerfCounters()`? Is that apart of the same system? If so, then can you include that as well in your clean up @sywhang?

Reply:
> `COUNTER_ONLY` may be a nop

`COUNTER_ONLY` is macro that turns into nop.

> I'll just submit a separate PR that cleans this stuff out later.

Thanks! All these should be deleted as part of this cleanup:
- `COUNTER_ONLY`
- `ENABLE_PERF_COUNTERS`
- `perfcounter*.*` files
- `PerfCountersSupportedBuild`

For now, do not remove `COUNTER_ONLY` and `ENABLE_PERF_COUNTERS` from GC. It would prevent sharing the source with the .NET Framework.

Reply:
Thanks @jkotas. I submitted https://github.com/dotnet/coreclr/pull/22742 to address this.